tests/gdn/test_prefill_state_indices.py

```
@@ -0,0 +1,175 @@
+        pytest.skip("state_indices GDN prefill path requires SM100/SM103 (Blackwell)")
+    cuda_major = int(torch.version.cuda.split(".")[0]) if torch.version.cuda else 0
+    if cuda_major < 13:
+        pytest.skip(f"SM100 GDN prefill requires CUDA 13+, got {torch.version.cuda}")
+
+
+def _make_inputs(seq_lens, H, D, dtype, device, seed):
+    torch.manual_seed(seed)
+    total = sum(seq_lens)
+    num_seqs = len(seq_lens)
+    cu_seqlens = torch.tensor(
+        [0] + list(torch.cumsum(torch.tensor(seq_lens), 0)),
```

Ruff (0.15.21)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)